packages/shared/src/lib/featureManagement.ts

```diff
@@ -21,6 +22,10 @@ export class Feature<T extends JSONValue> {
 
 const feature = {
   feedVersion: new Feature('feed_version', 15),
+  onboardingAnimation: new Feature(
+    'onboarding_animation',
+    OnboardingAnimation.Control,
+  ),
```

Comment:
I think we missed this comment of mine 😬 https://github.com/dailydotdev/apps/pull/2736#pullrequestreview-1918945730. Would u mind changing the naming and values, else I can delete and create a new one in growthbook as u've created this.

Reply:
I believe versions are mostly valuable only on a/b testings, when we use them usually as follow up experiments doesn't make much sense as we set the winning version as default (not using the feature at all) and then the other as the next variation - which can be fine just as a true boolean value.
Plus this is a micro experiment so I don't think we have anything we need to relate with versions

Reply:
Checking with product just in case, anyways we should be ok either way.

Reply:
I mean this is an a/b testing, animations can easily be made, which means there can be follow-up experiments to this if they suddenly think of something new. But sure, let's go for boolean.

Reply:
I don't know, don't think we use versions for most of them in their full potential. But yeah this doubt I think is also coming from not having any real guidance on how we do these. 🙌 Thanks for resolving

Reply:
Agree with that, we can maybe write a guideline in the future on how we define variables based on certain assessments.